conda/joint-calling/meta.yaml

```diff
@@ -1,23 +1,23 @@
 {% set data=load_setup_py_data() %}
 
 package:
-  name: cpg-qc
+  name: joint-calling
   version: {{ data['version'] }}
 
 source:
   path: ../../
 
 build:
-  number: 3
+  number: 0
   noarch: python
 
 requirements:
   host:
     - python ==3.7.*
-    - hail >=0.2.61
+    - hail >=0.2.64.dev201e81ff56c5
```

Comment:
Will this break when `0.2.64.dev201e81ff56c5` gets deleted?

Reply:
Good point. The pin is pretty much unnecessary anyway.